packages/flutter_tools/test/host_cross_arch.shard/ios_content_validation_test.dart

```diff
@@ -47,6 +71,7 @@ void main() {
     });
 
     tearDownAll(() {
+      hiddenFile.deleteSync();
```

Comment:
do we need to gracefully handle a failure here, or is that not an issue on mac?

Reply:
I'll swap to `tryToDelete` to match the next line.